src/main/scripts/org/reaktivity/specification/nukleus/amqp/streams/send.to.client.when.fragmented/server.rpt

```diff
@@ -0,0 +1,55 @@
+#
+# Copyright 2016-2020 The Reaktivity Project
+#
+# The Reaktivity Project licenses this file to you under the Apache License,
+# version 2.0 (the "License"); you may not use this file except in compliance
+# with the License. You may obtain a copy of the License at:
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+
+property serverTransport "nukleus://streams/target#0"
+property message600 ${amqp:randomBytes(600)}
+
+accept ${serverTransport}
+  option nukleus:window 8192
+  option nukleus:transmission "duplex"
+
+accepted
+
+read nukleus:begin.ext ${amqp:beginEx()
+                             .typeId(nukleus:id("amqp"))
+                             .address("clients")
+                             .capabilities("RECEIVE_ONLY")
+                             .senderSettleMode("SETTLED")
+                             .receiverSettleMode("FIRST")
+                             .build()}
+
+write nukleus:begin.ext ${amqp:beginEx()
+                              .typeId(nukleus:id("amqp"))
+                              .address("clients")
+                              .capabilities("RECEIVE_ONLY")
+                              .senderSettleMode("SETTLED")
+                              .receiverSettleMode("FIRST")
+                              .build()}
+
+connected
+
+write nukleus:data.ext ${amqp:dataEx()
+                             .typeId(nukleus:id("amqp"))
+                             .deferred(600 - 155)
+                             .deliveryId(0)
+                             .deliveryTag("0")
+                             .messageFormat(0)
+                             .flags("SETTLED")
+                             .build()}
+
+write ${message600}
```

Comment:
We can call the `amqp:randomBytes(600)` EL function directly here, instead of defining a separate `message600` property.

This feedback applies to other similar scenario scripts too.